com.tercad.fingrom.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.tercad.fingrom
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: fingrom
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
```

Comment:
Why do you need full home access?

Reply:
Only to ".terCAD"-folder to store applications' data.

Reply:
Add `--persist=.terCAD` and remove it. This will create the folder relative to real home inside the per app sandbox folder. Ideally you should follow the xdg standard and use XDG_* locations to store app data.

Reply:
Hm.. not related to that access. Updated main thread.

Reply:
Resolved. Removed `--filesystem=home`, `persist` is not needed as well. Thanks